.github/renovate.json

```diff
@@ -43,6 +43,19 @@
       "datasourceTemplate": "github-tags",
       "depNameTemplate": "cert-manager/cert-manager",
       "extractVersionTemplate": "^(?<version>.*)$"
+    },
+    {
+      "customType": "regex",
+      "description": "Bump thanos version in the plugindefinition",
+      "fileMatch": [
+        "^thanos.charts.Chart.yaml$"
+      ],
+      "matchStrings": [
+        "appVersion:\\s*(?<currentValue>v?(?:\\d+\\.){0,2}\\d+)"
```

Comment:
What are you trying to achieve? Do you want to bump the appVersion of Thanos when there is a new release in [thanos-io/thanos](https://github.com/thanos-io/thanos/releases)?